core/workflow-core/src/main/scala/edu/uci/ics/amber/core/storage/result/iceberg/TableStatistics.scala

```diff
@@ -0,0 +1,139 @@
+package edu.uci.ics.amber.core.storage.result.iceberg
```

Comment:
move this class to `core.storage.result`. This class should be independent with Iceberg (or any other storage implementation)